flake.nix

```diff
@@ -7,6 +7,28 @@
 
   outputs = { self, nixpkgs, utils }: utils.lib.eachDefaultSystem (system: let
     pkgs = import nixpkgs { inherit system; };
+    # TODO: need to be moved as flake input once this repository is a flake
+    gitignore = import (builtins.fetchTarball {
```

Comment:
I don't think we can use `fetchTarball` on Hydra.

Reply:
This is correct (excepting if `allowed_uris` is set). 

I've planned to send a PR to the `hercules-ci/gitignore` to add the flake support.

Is it a short term issue? 
In this case, we can wait until the `gitignore` flake support is merged to merge this PR: I could then put this lib as a flake input.

Reply:
I finally switch to the `poetry2nix.cleanPythonSources` which works with the restricted evaluation.